### ISSUE TITLE
Creating new line items via uploadlineitems

### ISSUE THREAD

**Issue by viniciusd** [googleads/googleads-bidmanager-examples#11]
Can the uploadlineitems create new line items? As far as I have tested, it is only managed to edit existing line items. However, I need to programatically create new ones

**Comment by hernanperalta** [comment]
Have you tried the [Display & Video 360 API](https://developers.google.com/display-video/api/guides/getting-started/overview)? It was released this year, and provides a LOT of features. Not to be confused with [DoubleClick Bid Manager API](https://developers.google.com/bid-manager/guides/getting-started-api)

**Comment by viniciusd** [comment]
Yep, @hernanperalta, we managed to get beta access to the API before it was released. Things are looking way better with DV360 API. Thanks
